tests/api/Team/PermissionTest.php

```diff
@@ -241,7 +241,7 @@ private function canAccessToList(User $user, Team $team)
         $I->seeResponseByJsonPathContainsJson(sprintf('$[?(@.type=="%s")]', DirectoryEnumType::DEFAULT), ['_links' => [
             'team_create_item' => [],
             'team_sort_list' => [],
-            'team_edit_list' => [],
+            //'team_edit_list' => [],
```

Comment:
May be delete this string with comment.

Reply:
I will revert this change after release.